# SPDX-License-Identifier: Apache-2.0
# Copyright (C) 2025 Intel Corporation
adbscan_sub_node:
  ros__parameters:
    Lidar_type: 2D
    Lidar_topic: scan
    Verbose: True
    subsample_ratio: 2.0
    x_filter_back: 2.0
    y_filter_left: 2.0
    y_filter_right: -2.0
    z_filter: -1.3
    Z_based_ground_removal: 1.0
    base: 2.24585885
    coeff_1: -0.350137
    coeff_2: -0.22432557
    scale_factor: 0.07
    init_tgt_loc: 0.5
    max_dist: 1.5
    min_dist: 0.5
    max_linear: 0.8
    max_angular: 0.8
    max_frame_blocked: 5
    tracking_radius: 0.2

adbscan_sub_w_gesture_node:
  ros__parameters:
    gesture_enable: True
    Lidar_type: 2D
    Lidar_topic: scan
    Verbose: True
    subsample_ratio: 2.0
    x_filter_back: 2.0
    y_filter_left: 2.0
    y_filter_right: -2.0
    z_filter: -1.3
    Z_based_ground_removal: 1.0
    base: 2.24585885
    coeff_1: -0.350137
    coeff_2: -0.22432557
    scale_factor: 0.07
    init_tgt_loc: 0.5
    max_dist: 1.5
    min_dist: 0.5
    max_linear: 0.8
    max_angular: 0.6
    max_frame_blocked: 5
    tracking_radius: 0.2

adbscan_sub_w_gesture_audio_node:
  ros__parameters:
    gesture_enable: True
    audio_enable: True
    Lidar_type: 2D
    Lidar_topic: scan
    Verbose: True
    subsample_ratio: 2.0
    x_filter_back: 2.0
    y_filter_left: 2.0
    y_filter_right: -2.0
    z_filter: -1.3
    Z_based_ground_removal: 1.0
    base: 2.24585885
    coeff_1: -0.350137
    coeff_2: -0.22432557
    scale_factor: 0.07
    init_tgt_loc: 0.25
    max_dist: 1.5
    min_dist: 0.25
    max_linear: 0.5
    max_angular: 0.25
    max_frame_blocked: 5
    tracking_radius: 0.27
